examples/flax/language-modeling/run_bart_dlm_flax.py

```diff
@@ -888,7 +894,7 @@ def eval_step(params, batch):
                 num_eval_samples = len(tokenized_datasets["validation"])
                 # Avoid using jax.numpy here in case of TPU training
                 eval_samples_idx = np.arange(num_eval_samples)
-                eval_batch_idx = generate_batch_splits(eval_samples_idx, eval_batch_size, drop_last=False)
+                eval_batch_idx = generate_batch_splits(eval_samples_idx, eval_batch_size)
```

Comment:
Is `drop_last=False` causing issues with `nan`? If so feel free to open a new issue and we can discuss a solution!

Reply:
@sanchit-gandhi Nice tip! Thank you very much.